config/core/configmaps/features.yaml

```diff
@@ -111,3 +111,8 @@ data:
     # 1. Enabled: enabling tag header based routing
     # 2. Disabled: disabling tag header based routing
     tag-header-based-routing: "disabled"
+
+    # This feature flags allows users to opt-out of the runtime /var/log mount
+    # requirement in the runtime contract
```

Comment:
It's been long enough that I forget the wording, but "requirement" seems strong.  I though this was a weaker word than MUST, like SHOULD?

Reply:
This breaks the v1 runtime contract since it's a MUST. We'll want to do a pass at creating a v2 where we drop which is on my TODO.